ng2-components/ng2-alfresco-datatable/src/components/datatable/datatable.component.ts

```diff
@@ -236,7 +236,8 @@ export class DataTableComponent implements AfterContentInit, OnChanges {
         let event = new DataCellEvent(row, col, []);
         this.showRowActionsMenu.emit(event);
 
-        return this.checkPermissions(row, event.value.actions);
+        return event.value.actions;
+        // return this.checkPermissions(row, event.value.actions);
```

Comment:
this code disables permissions functionality until we find out how to fix it. The intention is to uncomment it back in a separate PR when we have solution